docs/App.fs

```diff
@@ -403,9 +407,14 @@ let main = React.memo(fun () ->
         ]
     ])
 
-let render' = React.memo(fun () ->
+[<ReactComponent>]
```

Comment:
In a future PR I'll update all the components to use this for function components